app/views/spotlight/sir_trevor/blocks/_oembed_block.html.erb

```diff
@@ -1,6 +1,10 @@
-<div class="content-block item-text">
-  <div class="oembed-block <%= "pull-#{ oembed_block.text_align }" if oembed_block.text.present? %>">
+<div class="content-block item-text clearfix">
```

Comment:
:+1: good catch.  Pushed the row class up.

###  Before
![before-row](https://cloud.githubusercontent.com/assets/96776/6629867/ba4b1c28-c8cf-11e4-8cc2-b43cc9b003c5.png)

### After
![after-row](https://cloud.githubusercontent.com/assets/96776/6629868/ba4b443c-c8cf-11e4-8992-57ef77dba6e5.png)